plat/marvell/a8k/a70x0_amc/plat_def.h

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Would prefer to rename this file to mvebu_def.h, because plat_def.h is too generic a name.